bazel/linting/scripts/lint_go.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sh
+
+
+#Golang setup
+export PATH="${PWD}/${GO_LOCATION}/bin:$PATH"
+export SOURCE_LINT="$(find $PWD -name ${GO_LIBRARY_NAME})"
+export GOPATH=$SOURCE_LINT
+echo "go path is " $GOPATH
+ls $GOPATH
+
+echo "running lint on directory $(find $PWD -name enfabrica)/enkit"
+#
+## this is necessary for cache + homdir lib errors
+export HOME="$PWD"
+mkdir -p .cache
+
+#
+#
+##create output files
+mkdir -p $(dirname ${LINT_OUTPUT})
+touch ${LINT_OUTPUT}
+export LINT_OUTPUT="$PWD/${LINT_OUTPUT}"
+
+#fetch list of changed files from genrule
+
+read -a arr <<< $(cat ${GIT_DATA})
+
+cd $(find $PWD -name enfabrica)/enkit
+echo $PWD
+#golangci-lint run ./...
+ls lib/khttp/protocol
+for i in "${arr[@]}"
+do
+  if [[ $i == *.go ]]; then
+    go_package=$(dirname $i)
+    golangci-lint run $go_package --issues-exit-code 0 2>&1 | sed 's/.*'enkit'//' | tee ${LINT_OUTPUT}
```

Comment:
so, in PR https://github.com/enfabrica/enkit/pull/29/files I see you pull in golangci-lint, so bazel can build it.

But in this PR, I don't see any code to pull in the golangci built by bazel? The line here is just running the golangci on your system, which won't be available on random cloud machines we run this test on?

The typical way to address this is:
1) Have a dependency in the rule to run the linter on the linting tool, eg, something like _linter = Label("@golangci-lint...").
2) Use the path of the bazel built golangci-lint in scripts, rather than the internal one.